app/controllers/tickets_controller.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+class TicketsController < ApplicationController
+  before_action :set_ticket, only: %i[show update destroy]
+
+  # GET /tickets
+  def index
+    @tickets = Ticket.all.map do |res|
+      render_ticket(res)
+    end
+    render json: @tickets
+  end
+
+  # GET /tickets/:ticket_id
+  def show
+    render json: render_ticket(@ticket)
+  end
+
+  # POST /tickets/
+  def create
+    @ticket = Ticket.create(ticket_params)
```

Comment:
Consider an extraction to use cases - at least for the first step - it would be super helpful to start Monte way refactorization :)